.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
+.history/
+.vscode/
```

Comment:
these ignores should be configured in your global `.gitignore`, don't add them here

Reply:
Thx. Done.